tests/__snapshots__/runner.test.js.snap

```diff
@@ -32,7 +32,7 @@ Array [
         "sauce:options": Object {
           "build": "buildname",
           "capturePerformance": true,
-          "crmuxdriverVersion": "beta",
+          "crmuxdriverVersion": "stable",
```

Comment:
We should add a test to ensure that we actually are able to change the crmuxdriver version.